src/xcode/ENA/ENA/Source/View Helpers/AccessibilityIdentifiers.swift

```diff
@@ -305,5 +305,27 @@ enum AccessibilityIdentifiers {
 	enum ThankYouScreen {
 		static let accImageDescription = "AppStrings.ThankYouScreen.accImageDescription"
 	}
+	
+	enum Statistics {
+		
+		enum Card {
+			
+			enum Infections {
+				static let title = "AppStrings.Statistics.Card.Infections.title"
+			}
+			
+			enum Incidence {
+				static let title = "AppStrings.Statistics.Card.Incidence.title"
+			}
+			
+			enum KeySubmissions {
+				static let title = "AppStrings.Statistics.Card.KeySubmissions.title"
+			}
+			
+			enum ReproductionNumber {
+				static let title = "AppStrings.Statistics.Card.ReproductionNumber.title"
+			}
+		}
+	}
```

Comment:
I personally don't see the need to be this specific for these cards.
```suggestion
        enum Statistics {
		static let Infections = "AppStrings.Statistics.Card.Infections.title"
		static let Incidence = "AppStrings.Statistics.Card.Incidence.title"
		static let KeySubmissions = "AppStrings.Statistics.Card.KeySubmissions.title"
		static let ReproductionNumber = "AppStrings.Statistics.Card.ReproductionNumber.title"
	}
```

Reply:
fixed.